Implementation/beer creation

BBR2394 opened the pull request:
here is the new beer activity

TheNoobProgrammeur committed:
:twisted_rightwards_arrows: Add IdUserImmage (#48)

Co-authored-by: Antoine <aberthier@capensis.fr>

BBR2394 committed:
:construction: making the create new beer activity

BBR2394 committed:
:construction: making the create new beer activity

BBR2394 committed:
:sparkles: i have added the cropp image lib according to what antoine had done on the test project it is almost done i need to print the picture in a circle and not in a square ...

BBR2394 committed:
:sparkles: i have added the cropp image lib according to what antoine had done on the test project it is almost done i need to print the picture in a circle and not in a square ...

BBR2394 committed:
:construction: now i have the image that can be taken and which is cropped coreclty, of course it was not so easy whith some screen problem...

BBR2394 committed:
:construction: now i have the image that can be taken and which is cropped coreclty, of course it was not so easy whith some screen problem...

BBR2394 committed:
:construction: update and continue to implement feature

i have update the gradle file for the different import of firabase; i am continuing the adding beer view; and i am going to send data to firebase

BBR2394 committed:
:construction::construction: progressing request firebase

i have done a request to firebase, i am pretty sure it works however i am not sure it is totally configure at firebase

BBR2394 committed:
:construction: i manage to a merge conflict on my own branch ...

BBR2394 committed:
:construction: fix problem with label

i had a color problem for the label

BBR2394 committed:
:construction: added a spinner

BBR2394 committed:
:construction: send a new beer totally

i have almost finished but i have to clean the code and check data, because if we validate whithout data, it crashs... and close corectly the activity

BBR2394 committed:
:sparkles: adding a new beer is done

i think i have finished this part, maybe some things can/could be changed but it works

TheNoobProgrammeur reviewed (commented):
Ca me semble bien. Simplment deux points :

- Il faudrais que tu enléve le bouton neew beer pour que ce soit le bouton ajouter qui active la fonction.
- Ne rend pas l'image obligatoire

BBR2394 committed:
:sparkles: last change and clean

i forget to remove the button to go to the activity, it was just for testing purpose. and the picture is no more mandatory

BBR2394 committed:
:sparkles: clean code MainActivity

i forgot to remove an unused function

BBR2394 committed:
:sparkles::robot: fix navigation

i manage to use the bottom navigation to open the other activity

TheNoobProgrammeur reviewed (approved):
C'est ok pour moi
